cmd/pinniped-proxy/Cargo.toml

```diff
@@ -17,4 +17,5 @@ log = "0.4"
 pretty_env_logger = "0.4"
 structopt = "0.3"
 # Update to 0.3 once apparent issue with the nascent 0.3.5 is fixed.
-tokio = { version = "0.2", features = ["full"] }
\ No newline at end of file
+tokio = { version = "0.2", features = ["full"] }
```

Comment:
So, apparently, you can decide which compilation options you want for a given package. Interesting!

Reply:
Yeah, and can therefore save compile time depending on what you need.